doc/src/stdlib/arrays.md

```diff
@@ -74,6 +74,7 @@ Base.flipdim
 Base.circshift
 Base.circshift!
 Base.circcopy!
+Base.contains(eq::Function, itr, x)
```

Comment:
It's only required if there's more than one docstring for `contains` listed in stdlib. `contains` already has another docstring listed at `stdlib/strings.md:34` so that one should also have it's signature narrowed to `Base.contains(::AbstractString, ::AbstractString)`.